get-started/debug.qmd

```diff
@@ -0,0 +1,232 @@
+---
+title: Debug, troubleshoot, & help
+
+filters:
+    - line-highlight
+---
+
+## Common issues
+
+Before jumping into general debugging techniques, lets cover some common issues that you may encounter when developing Shiny applications, and explain why they happen.
+
+### Missing output
+
+Sometimes, output won't appear at all.
+This most commonly happens when an output reads a non-existent input, for example:
+
+```{shinylive-python}
+# | standalone: true
+# | components: [editor, viewer]
+# | layout: vertical
+# | viewerHeight: 120
+
+from shiny.express import input, render, ui
+
+ui.input_slider("val", "Slider value", min=0, max=10, value=5)
+
+
+# Nothing renders because input.wrong_id() doesn't exist!
+@render.text
+def slider_val():
+    return f"Slider value: {input.wrong_id()}"
+
+
+## file: fixed.py
+from shiny.express import input, render, ui
```

Comment:
We need a comment here like "copy this solution" or something that indicates that only `app.py` is run by shinylive.

https://github.com/posit-dev/py-shiny-site/pull/261#issuecomment-2822425919